Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
this is wrong on the interface. The same instance cannot be documented as ``Config``